Update reset password information (#3609)
w3f/polkadot-wiki: docs/learn/learn-account-generation.md

Note that this password does NOT protect your seed phrase. If someone knows the twelve words in your
mnemonic seed, they still have control over your account even if they do not know the password.

### Reset Password

:::warning

Before following the instructions below make sure you have your mnemonic phrase stored in a safe place accessible to you.

:::

> Let's say you created `ACCOUNT 1` protected by password `PSW 1`, and you also exported the JSON file `JSON 1`. To reset the passowrd of your polkadot `ACCOUNT 1` using the browser extension you must follow the following steps:

Reviewer: Keeping it simple by not mentioning about JSON file export here

```suggestion
Let's say you created `ACCOUNT 1` protected by password `PSW 1`. To reset the password of your `ACCOUNT 1` using the browser extension you must follow the following steps:
```

- On the browser extension go to `ACCOUNT 1` and click "Forget account". This action will delete the access to your account. Note that your tokens are still in your account on the polkadot blockchain network.
- On the browser extension click the "+" button in the topright corner and select the option "Import account from pre-existing seed". After entering the mnemonic phrase you can chose a new password `PSW 2`.

:::info JSON files do not allow to change account passwords

> If you add the account to the extension using the option "Restore account from backup JSON file", this will allow you to restore access to your account using JSON file `JSON 1` protected by the password `PSW 1`, but not to set a new password. Thus, `PSW 1` will become the account password by default.

Reviewer: ```suggestion
If you add the account to the extension using the option "Restore account from backup JSON file", this will allow you to restore access to your account using JSON file protected by the password `PSW 1`, but does not let you set a new password. Thus, `PSW 1` will become the account password by default.
```

:::

> :::info Cold wallets do not need passwords

Reviewer: ```suggestion
:::info Accounts on Cold wallets do not need passwords
```

> For hardware wallets such as [Ledger](https://www.ledger.com/) you do not need to set a password for your account. Each time you need to access funds in your account you are required to sign using your Ledger device. Also, Ledger wallets are deterministic wallets, meaning that you can generate multiple accounts for for multiple blockchain networks without having different passwords to access such accounts.

Reviewer: ```suggestion
For hardware wallets such as [Ledger](https://www.ledger.com/), you may have to set a PIN for accessing the accounts on the device, but you do not need to set a password for every individual account. When you need to make transactions with your account, you are required to sign using your Ledger device. Also, Ledger wallets let you generate multiple accounts for multiple blockchain networks without having to set different passwords to access such accounts.

```

You can use this backup file to restore your account. This backup file is not readable unless it is
decrypted with the password.

### Reset password

To reset the password of an account created with Polkadot-JS you just need to go in the "Accounts" tab, click the icon with three vertical dots on your account and select "Change this account's password".

:::info

> If you create an account first using Polkadot-JS, and then you add it to the browser extension, to change the passowrd of such account you need to follow the [guidelines for the browser extension](#reset-password).

Reviewer: This section is named as reset-password as well. The link works and points to the reset-password subsection from the JS Extension section. In general, the recommendation is to give unique titles to sections and subsections

Author: I specified different subtitles. See latest push.

You can use this backup file to restore your account. This backup file is not readable unless it is
decrypted with the password.

### Reset password using Polkadot-JS

To reset the password of an account created with Polkadot-JS you just need to go in the "Accounts" tab, click the icon with three vertical dots on your account and select "Change this account's password".

:::info

> If you create an account first using Polkadot-JS, and then you add it to the browser extension, to change the passowrd of such account you need to follow the [guidelines for the browser extension](#reset-password).

Reviewer: Polkadot-JS usually refers to the entire JS stack. This is referring to the Apps UI. Also fixed typo of password

```suggestion
If you create an account first using Polkadot-JS Apps UI, and then add it to the browser extension, you need to follow the [guidelines for the browser extension](#reset-password) to change the password of such account.
```

You can use this backup file to restore your account. This backup file is not readable unless it is
decrypted with the password.

> ### Reset password using Polkadot-JS

Reviewer: ```suggestion
### Reset password using Polkadot-JS Apps UI
```

To reset the password of an account created with Polkadot-JS you just need to go in the "Accounts"
tab, click the icon with three vertical dots on your account and select "Change this account's
password".

:::info

> If you create an account first using Polkadot-JS Apps UI, and then add it to the browser extension, you need to follow the [guidelines for the browser extension](#reset-password) to change the password of such account.

Reviewer: The markdown link needs to be updated

Author: Done
